.github/workflows/ci.yml

```diff
@@ -102,6 +102,12 @@ jobs:
         uses: buildpacks/github-actions/setup-pack@v5.7.2
       - name: Pull builder image
         run: docker pull ${{ env.INTEGRATION_TEST_CNB_BUILDER }}
+      - name: Check for tests that start the container
+        id: needs-run-image
+        run: echo "result=$(grep --quiet --recursive "start_container" ${{ matrix.dir }}/tests && echo "true" || echo "false")" >> $GITHUB_OUTPUT
+      - name: Pull run image
+        if: steps.needs-run-image.outputs.result == 'true'
+        run: docker pull $(pack builder inspect ${{ env.INTEGRATION_TEST_CNB_BUILDER }} -o json | jq -r '.remote_info.run_images[0].name')
```

Comment:
(optional) The `pack builder inspect` command currently generates metadata for both the local builder image (if it exists) and also the remote builder image on the registry. This is a slightly puzzling implementation choice on Pack CLI's side. There doesn't appear to be a way to disable the remote image querying.

As such, using `pack builder inspect` results in additional Docker Hub calls compared to using `docker inspect`. Since these will only be once per CI job in this step (rather than once per `libcnb-test` integration test) then the impact could be worse - however, for Python I decided to use `docker inspect` instead:
https://github.com/heroku/buildpacks-python/pull/223

I don't mind which you use in this repo though - up to you :-)

Reply:
Yeah, I saw your PR and will adopt the same technique. As always, you find the optimal way @edmorley 😄

Reply:
Fixed to use `docker inspect`